launchable/test_runners/minitest.py

```diff
@@ -0,0 +1,39 @@
+import click
+import os, sys
+from . import launchable
+
+
+@click.argument('files', required=True, nargs=-1)
+@launchable.optimize.tests
+def optimize_tests(client, files):
+    def parse(fname:str):
+        '''
+        Scan a file, directory full of *.rb, or @FILE
+        '''
+        if os.path.isdir(fname):
+            client.scan(fname, '**/*.rb')
+        elif fname=='@-':
```

Comment:
We need to note the behavior to `help` prop in the `@click.argument()` annotation.